cmd/swapcli/main.go

```diff
@@ -549,17 +549,17 @@ func runMake(ctx *cli.Context) error {
 
 	c := newRRPClient(ctx)
 
-	relayerEndpoint := ctx.String(flagRelayerEndpoint)
-	relayerFee := new(apd.Decimal)
-	if relayerEndpoint != "" {
-		if relayerFee, err = cliutil.ReadUnsignedDecimalFlag(ctx, flagRelayerFee); err != nil {
+	// In the near future, the relayer will only be used if Bob doesn't have funds to claim,
+	// but in the current version, it is always used whenever this relayerFee is non-nil.
+	var relayerFee *apd.Decimal
+	if ctx.IsSet(flagRelayerFee) {
+		relayerFee, err = cliutil.ReadUnsignedDecimalFlag(ctx, flagRelayerFee)
+		if err != nil {
 			return err
 		}
-	} else if ctx.IsSet(flagRelayerFee) {
-		return errMustSetRelayerEndpoint
-	}
-	if relayerFee.Cmp(apd.New(1, 0)) > 0 {
-		return errRelayerFeeTooHigh
+		if relayerFee.Cmp(minRelayerFee) < 0 || relayerFee.Cmp(maxRelayerFee) > 0 {
```

Comment:
why do we need the `rrelayerFee >= minRelayerFee` check? a relayer could still manually set a lower fee, I think we should warn users potentially but not reject

Reply:
also related to #324

Reply:
ah I see you removed any way for the relayer to configure their fee.

Reply:
Yeah, if we wanted to make the relayer fee configurable, we'd need to do it in conjunction with https://github.com/AthanorLabs/atomic-swap/issues/324 at the same time. Otherwise, it is an anti-feature in the sense that would increase failed swaps.